packages/safety-suite-api/src/api/dashboardSink/types.ts

```diff
@@ -25,3 +27,21 @@ export interface DashboardBaseFields {
   recordNumber: number;
   groupId: number;
 }
+
+export interface KpiTimeFrame {
+  from: string;
+  to: string;
+}
+
+export interface KpiTimeQuery {
+  field: string;
+  frame?: KpiTimeFrame;
+  interval?: string;
+}
+export interface KpiQuery {
+  customerAlias: string;
+  metric: string;
+  aggregation?: AggregateFunctionLiteral;
```

Comment:
Is this optional?  I think it should be required.  I guess the server could implement a default aggregation for each metric, but it isn't something we've talked about yet.